pkgs/development/libraries/at-spi2-core/default.nix

```diff
@@ -30,11 +32,13 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [
+    glib
```

Comment:
why glib in here and in tracker

Reply:
```
at-spi2-core> WARNING: Could not detect glib version, assuming 2.54. You may get build errors if your glib is older.
at-spi2-core> Program glib-genmarshal found: NO
at-spi2-core>
at-spi2-core> atspi/meson.build:76:0: ERROR: Program 'glib-genmarshal' not found or not executable
```

Reply:
can't repro `nix build ".#pkgsCross.aarch64-multiplatform.at-spi2-core" ".#at-spi2-core"`

Reply:
though i think thats because gir(and maybe something else) is propagating glib

Reply:
Exactly.  It's required when introspection is disabled, which is why it's needed now and not before.